client/matchmaking/admin-matchmaking-times.tsx

```diff
@@ -0,0 +1,658 @@
+import React, { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+import { ReadonlyDeep } from 'type-fest'
+import {
+  ALL_MATCHMAKING_TYPES,
+  MatchmakingType,
+  matchmakingTypeToLabel,
+} from '../../common/matchmaking'
+import {
+  AddMatchmakingTimeRequest,
+  GetFutureMatchmakingTimesResponse,
+  GetMatchmakingTimesResponse,
+  GetPastMatchmakingTimesResponse,
+  MatchmakingTimeJson,
+} from '../../common/matchmaking/matchmaking-times'
+import { apiUrl } from '../../common/urls'
+import { ThunkAction } from '../dispatch-registry'
+import { longTimestamp } from '../i18n/date-formats'
+import { MaterialIcon } from '../icons/material/material-icon'
+import { ElevatedButton, TextButton } from '../material/button'
+import { CheckBox } from '../material/check-box'
+import { TabItem, Tabs } from '../material/tabs'
+import { abortableThunk, RequestHandlingSpec } from '../network/abortable-thunk'
+import { encodeBodyAsParams, fetchJson } from '../network/fetch'
+import { LoadingDotsArea } from '../progress/dots'
+import { useImmerState } from '../react/state-hooks'
+import { useAppDispatch } from '../redux-hooks'
+import { externalShowSnackbar } from '../snackbars/snackbar-controller-registry'
+import { CenteredContentContainer } from '../styles/centered-container'
+import { styledWithAttrs } from '../styles/styled-with-attrs'
+import { bodyLarge, bodyMedium, titleLarge } from '../styles/typography'
+
+function getMatchmakingTimesHistory(
```

Comment:
This naming seems weird since we also have `getMatchmakingTimesPast`, maybe it should just be called `getMatchmakingTimes`?